llvm/test/CodeGen/RISCV/GlobalISel/legalizer/merge-unmerge-d.mir

```diff
@@ -0,0 +1,38 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+# RUN: llc -mtriple=riscv32 -mattr=+d -run-pass=legalizer %s -o - \
```

Comment:
This should be in the rv32 directory

Reply:
Didn't you add 0154e53bf307aa501a6d9d6a1bafa6a6fcb2a7ad?